src/range.coffee

```diff
@@ -189,13 +189,33 @@ class Range.NormalizedRange
   #
   # Returns updated self or null.
   limit: (bounds) ->
-    nodes = $.grep this.textNodes(), (node) ->
-      node.parentNode == bounds or $.contains(bounds, node.parentNode)
+    if @commonAncestor == bounds or $.contains(bounds, @commonAncestor)
```

Comment:
I've not written CoffeeScript before yesterday, but [the docs](http://coffeescript.org/#operators) suggest that `==` in CoffeeScript compiles to `===` in JS.